pkg/api/v1alpha1/cluster_webhook.go

```diff
@@ -113,11 +120,56 @@ func (r *Cluster) ValidateUpdate(old runtime.Object) error {
 	return nil
 }
 
+func validateEksaVersionSkew(new, oldCluster *Cluster) field.ErrorList {
+	var allErrs field.ErrorList
+	eksaVersionPath := field.NewPath("spec").Child("EksaVersion")
+
+	if new.Spec.EksaVersion != nil && oldCluster.Spec.EksaVersion != nil {
+		parsedClusterVersion, err := semver.New(string(*oldCluster.Spec.EksaVersion))
+		if err != nil {
+			return allErrs
```

Comment:
If the old version is invalid and they're trying to change it, does it matter? I don't think we should block an upgrade because the old value is invalid.